docs/source/getting-started/installing.adoc

```diff
@@ -0,0 +1,128 @@
+[[installing-minishift]]
+= Installing Minishift
+:icons:
+:toc: macro
+:toc-title:
+:toclevels: 1
+
+toc::[]
+
+The following section describes how to install Minishift and the required dependencies.
+
+[[install-prerequisites]]
+== Prerequisites
+
+Minishift requires a hypervisor to start the virtual machine containing
+OpenShift. Make sure that the hypervisor of your choice is installed and
```

Comment:
The virtual machines does not contain OpenShift. But it provisions OpenShift in the VM using "oc cluster up" . So I think the text should be `Minishift requires a hypervisor to start a virtual machine and it is used to provision single node OpenShift cluster.`